equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/repository/Crm14PSEMessageRepository.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.data.Crm14PSEMessagesModel;
+
+import java.util.List;
+
+@Repository
+public interface Crm14PSEMessageRepository extends JpaRepository<Crm14PSEMessagesModel, Long>{
+    List<Crm14PSEMessagesModel> findByUSN(long usn);
```

Comment:
I think in this case, we should use `findAllByUsn()`. because the return is a list. By standard `findByUsn` usually refers to find one. But in this case USN is a FK and not PK.

Reply:
done